includes/Hooks.php

```diff
@@ -65,7 +65,7 @@ public static function getLanguageSelectorBoxHtml() {
 		$ret .= '
 			      </div>
 			      <div class="modal-footer">
-			        <button type="button" class="btn btn-default" data-dismiss="modal">'.wfMessage('cancel')->plain().'</button>
+			        <button type="button" class="btn btn-default" id="cancel" data-dismiss="modal">'.wfMessage('cancel')->plain().'</button>
```

Comment:
ca sert à quoi le id="cancel" ?

c'est beaucoup trop générique pour un nom d'id

Reply:
C'était utile quand je forçais l'apparition de la popup de choix des langues mais c'est plus utile, j'ai oublié de l'enlever

Reply:
tu peux le supprimer du coup ? c'est vraiment pas top de garder de id sur les balise html, surtout s'ils servent plus :)

Reply:
Oui je suis en train de le faire